### ISSUE TITLE
QC graph bug from 2.10 fixed, but new bug in 2.11?

### ISSUE THREAD

**Issue by hwick** [nservant/HiC-Pro#205]
Hello,
The legend labels from v2.11.0 appear to be mislabeled:
QC graph from v2.11.0:
![image](https://user-images.githubusercontent.com/33556230/48570240-e3a74800-e8d1-11e8-839b-4f4936096f3c.png)

For reference, here is the same sample analyzed from v2.10:
![image](https://user-images.githubusercontent.com/33556230/48570218-d5592c00-e8d1-11e8-858a-14c8de991368.png)
The numbers on the graph in this one are switched, but the colors in the legend seem to make sense at least.

Is there a difference between v2.11.0 and v2.11.1? 2.11.1 came out a day or so after I downloaded 2.11.0 but the only difference I could see between the versions was that "2.11.0beta" was changed to "2.11.1" in the changelog. Does 2.11.1 fix this issue?

**Comment by nservant** [comment]
Hi,
Thanks for reporting this issue.
Actually, this issue is more linked to the R and ggplot version you are using.
Could you please check them and tell me which versions you have ?

Then, plots in 2.10 are not correct. So I guess you are using a recent version of ggplot. That's why I updated the code in 2.11. In this version, % are good, but I will double check the legend.
Finally, since v.2.11, HiC-Pro is now compatible with multiQC.
I really advice to use multiQC ... In the future, I think that the R plots will be completely replaced by multiQC.
N

**Comment by hwick** [comment]
If I load R using default module, and then load ggplot2, here is the session info:
```
> sessionInfo()
R version 3.3.1 (2016-06-21)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Red Hat Enterprise Linux Server release 6.7 (Santiago)

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ggplot2_2.2.1

loaded via a namespace (and not attached):
 [1] colorspace_1.3-2 scales_0.5.0     lazyeval_0.2.1   plyr_1.8.4      
 [5] pillar_1.2.1     gtable_0.2.0     tibble_1.4.2     Rcpp_0.12.14    
 [9] grid_3.3.1       rlang_0.2.0      munsell_0.4.3   
```
We do have a more recent version of R (3.5.1), but I didn't see a way to specify a path to R in the config file, is there a way to do that? Would that solve the issue
When you say to use multiQC, do you mean to run it separately outside of HiC-Pro, or is there a way to specify to use that within HiC-Pro?
Thanks so much
~ Heather

**Comment by nservant** [comment]
For R, in the installation folder, you should have a config-system.txt file.
If you update the R_PATH to 3.5.1 it should work. However, be sure that all packages are installed (otherwise, you will have some errors)

For MultiQC, you need to install the last version 1.6.
Then, go into the HiCPro output folder and run 
`multiqc .`
As MultiQC will scan for all files in the current "." directory, I would advice to run it within the stats folder. It would be much faster and all the files it need are in this folder.
Best

**Comment by nservant** [comment]
Fixed in devel 2.11.3
